manim/mobject/text/code_mobject.py

```diff
@@ -156,7 +156,8 @@ def construct(self):
     # For more information about pygments.lexers visit https://pygments.org/docs/lexers/
     # from pygments.lexers import get_all_lexers
     # all_lexers = get_all_lexers()
-    styles_list = list(get_all_styles())
+    # styles_list = list(get_all_styles())
+    _styles_list_cache = None
```

Comment:
```suggestion
    _styles_list_cache: list[str] | None = None
```
Always helps to keep a nicely typed library!

Reply:
Sounds good!